python/vast/voidfinder/postprocessing.py

```diff
@@ -725,7 +733,7 @@ def save_output_from_find_voids(
     primaryHDU.header['WALLNUM'] = (wallHDU.header['WALLNUM'], 'Wall Galaxy Count')
     primaryHDU.header['FIELDNUM'] = (fieldHDU.header['FIELDNUM'], 'Field Galaxy Count')
     primaryHDU.header['DENSITY'] = (mknum(num_gals/vol), 'Galaxy Count Density (Mpc/h)^-3')
-    primaryHDU.header['AVSEP'] = (mknum(np.power(vol/num_gals, 1/3)), 'Average Galaxy Separation (Mpc/h)')
+    if num_gals > 0: primaryHDU.header['AVSEP'] = (mknum(np.power(vol/num_gals, 1/3)), 'Average Galaxy Separation (Mpc/h)')
```

Comment:
Cosmetic comment: it's a bad idea to have conditionals all on one line, unless using the pythonic syntax `x = y if z else ...`, because it makes code less readable. Instead, write as a conditional block:
```
if num_gals > 0:
    primaryHDU.header['AVSEP'] = ...
```